main.go

```diff
@@ -6,24 +6,34 @@ import (
 	"log"
 	"time"
 
+	"github.com/libp2p/go-libp2p-core/crypto"
+	peer "github.com/libp2p/go-libp2p-core/peer"
 	id "github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	"github.com/libp2p/hydra-booster/httpapi"
+	"github.com/libp2p/hydra-booster/hydra"
 	"github.com/libp2p/hydra-booster/metrics"
-	"github.com/libp2p/hydra-booster/runner"
+	"github.com/libp2p/hydra-booster/ui"
+	uiopts "github.com/libp2p/hydra-booster/ui/opts"
 	"github.com/libp2p/hydra-booster/utils"
 )
 
-const defaultKValue = 20
+const (
+	defaultKValue = 20
+	httpAPIAddr   = "127.0.0.1:7779"
```

Comment:
Can you add a log that prints this endpoint on start?

Reply:
Yes, good idea - added in https://github.com/libp2p/hydra-booster/pull/35